### ISSUE TITLE
Possess (Release 1.0 version) Issue

### ISSUE THREAD

**Issue by StrangeOne101** [StrangeOne101/Spirits#1]
# Description
Possess can cause the following error. This is on the `release` branch as the `refactor` branch has a new Possess that is unfinished.

```
[?:?] [87:26:16 WARN]: [Projectkorra] Task #96 for ProjectKorra v1.16.2 generated an exception java.lang.IllegalArgumentException: Player must be in spectator mode
        at com.google.common.base.Preconditions.checkargument(Preconditions.java:145) -[guava-31.8.1-jre.jar:?]
        at org.bukkit.craftbukkit.v1_18_82.entity.CraftPlayer.setSpectatorTarget(CraftFlayer.java:2248) -[pufferfish-1.18.2.jar:git-Pufferfish-68]
        at me.numin.spirits.ability.spirit.Possess.remove(Possess.java:178) -[Spirits-1.6-PK1.16.6.jar:?]
        at com.projectkorra.projectkorra.ability.CoreAbility.progressAll(CoreAbility.java:263) -[Projectkorra-1.16.2.jar:fl at com.projectkorra.projectkorra.BendingManager.run(BendingManager.java:117) -[Projectkorra-1.16.2.jar:?]
        at org.bukkit.craftbukkit.v1_18_82.scheduler.CraftTask.run(CraftTask.java:181) -[pufferfish-1.18.2.jar:git-Pufferfish-68]
```

**Comment by Kingie07** [comment]
Can confirm latest version possess errors and crashes all of bending
https://justpaste.it/5hbft

**Comment by StrangeOne101** [comment]
Fixed as of f12e7bf7176ccaad2524229f0d9576c091444fc3
